### PR TITLE
Remove duplicate Context and Problem Fix sections

### PR DESCRIPTION
**Pull request by mccordgh**
## Description
There is a duplicate "Context" section in the issue template. 
There is an unnecessary "Problem Fix" section in the issue template.
I have removed both sections.

## Motivation and Context
Required because of unnecessary repetition and it being bad practice to suggest code.

## Checklist:
 Go over all the following points, and put an \`x\` in all the boxes that apply. 
 If you're unsure about any of these, don't hesitate to ask. We're here to help! 
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,9 +9,6 @@ Tell us what should happen
 ## Actual Behavior
 Tell us what happens instead
 
-## Possible Fix
```
Comment (asimonia) [inline ISSUE_TEMPLATE.md:12]: Good idea to leave this out!

**Review by nchemsak — APPROVED** [review]
👍

**Review by rtwhitfield84 — APPROVED** [review]
Ⓜ️ 👍

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -20,9 +17,6 @@ reproduce this bug include code to reproduce, if relevant
 3.
 4.
 
-## Context
```
Comment (asimonia) [inline ISSUE_TEMPLATE.md:23]: I see what you did here :1st_place_medal:

**Pull request merged**